src/starknet-wrappers.ts

```diff
@@ -103,6 +132,25 @@ export abstract class StarknetWrapper {
     }
 
     public abstract getTxStatus(options: GetTxStatusOptions): Promise<ProcessResult>;
+
+    protected prepareDeployAccountOptions(options: DeployAccountOptions): string {
+        const wallet = options.wallet? "'" + options.wallet + "'" : "None";
+        const accountName = options.accountName? "'" + options.accountName + "'" : "'__default__'";
+        const accountDir = options.accountDir? "'" + options.accountDir + "'" : "None";
+        return "Namespace(" +
```

Comment:
I'm almost sure this can be written in a better way, we can discuss it. Basically the idea of `prepareXOptions` was to prepare CLI options for the StarknetCLI commands that are being wrapped.